### ISSUE TITLE
Get inferred triples as graph

### ISSUE THREAD

**Issue by constnick** [mdesalvo/RDFSharp#152]
Hi Marco,
is it possible to get an RDFGraph containing inferred triples from RDFOntologyReasonerReport?
And to get an RDFTriple from RDFOntologyReasonerEvidence?
Thanks!

**Comment by mdesalvo** [comment]
Sure. May it be like this?
```
RDFGraph inferenceGraph = reasonerReport.ToRDFGraph();
RDFTriple inferenceTriple = reasonerEvidence.ToRDFTriple();
```

**Comment by constnick** [comment]
Yes, this way

**Comment by mdesalvo** [comment]
Will be available in next release (2.20)
